library/x509_crt.c

```diff
@@ -61,6 +61,7 @@
 #if defined(_WIN32) && !defined(EFIX64) && !defined(EFI32)
 #define WIN32_LEAN_AND_MEAN
 #include <windows.h>
+#include <intsafe.h>
```

Comment:
do we need this change :) or can't we bear the thought of another update? :)

Reply:
Let's see if it clears CI, and I could try to push a last commit to remove that. Re reviewing an include could be simple.

Reply:
(note that the only functional changes to this file now are the #if changes)